check_me_out_but_better.cpp

```diff
@@ -0,0 +1,11 @@
+#include <iostream>
+#include <string>
+#include <math.h>
+using namespace std;
+
+int main{
+    int a, int c;
```

Comment:
Olha não precisa do segundo int.
```suggestion
    int a, c;
```